build.js

```diff
@@ -278,8 +278,8 @@ function getSource (callback) {
         },
         banner: {
           visible: false,
```

Comment:
Sorry, didn't spot this but I guess this should have been flipped to `true`?

Reply:
https://github.com/nodejs/nodejs.org/pull/3361

Reply:
> Sorry, didn't spot this but I guess this should have been flipped to `true`?